### DISCUSSION TITLE
Feature Request: Notes

### DISCUSSION THREAD

**Discussion by SquidwardHill** [americanexpress/one-app#810] [Ideas]
Hey there,

Thanks for hard work on the app!

I would love the ability to attach notes to:

- automatic payments
- individual payments
- transactions

**Context:** I'm sharing an account with a family member and it will give us more historical context about what certain payments are for, or what the automatic transaction covers for our own record keeping/accounting. 

Beyond adding a note for an individual payment, _a really neat feature would be:_ 
Ability to attach a related transaction id (or multiple) to a payment (where the UI shows more information about the transaction so that it's meaningful to the customer). This could extend to automatic payments as well.

**Comment by 10xLaCroixDrinker** [comment]
Hi Sydney,
 
Thank you for your post.  We appreciate you taking the time to contact us.  Please know that this platform is intended for One App developers.  Based on your message, it appears that you have customer feedback regarding our mobile application.  If that is the case, you may direct that correspondence to the appropriate team using the app’s feedback function. Within the app, please navigate to "Account" -> "Contact Us" -> "Leave Feedback."

Please know that we do not accept submissions of ideas related to potential new or future products or services from individuals external to American Express, so please limit your message to general feedback related to our mobile application.

Thank you for thinking of American Express.
